src/DroneSafetyApi/Models/HeatmapsQuery.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Azure.Documents.Spatial;
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Guessing this should be `Microsoft.Azure.Documents.Linq` so that you can talk to DocDB (this is for SQL I think).